features/aws-ids.yaml

```diff
@@ -1,3 +1,6 @@
+xenial: ami-06e94647aeaed1e5c
+xenial-fips: ami-077e4c339a098fc9f
```

Comment:
Since you are adding the fips images on the same file, we need to update the `tools/refresh-aws-pro-ids` script to not override the fips ids here

Reply:
yeah, just updating the releases refreshed alone.